MapboxNavigation/FeedbackItem.swift

```diff
@@ -7,7 +7,7 @@ extension UIImage {
     }
 }
 
-struct FeedbackItem {
+public struct FeedbackItem {
```

Comment:
Note that a Swift struct can’t bridge to Objective-C, nor can any method or property that uses that type.